### PR TITLE
Uplift polkadot v0.9.36

### PR DESCRIPTION
**Pull request by abhijeetbhagat**

### CODE REVIEW

File: pallets/airdrop/src/lib.rs [hunk 1]
```diff
@@ -234,6 +233,7 @@ pub mod pallet {
 	impl<T: Config> Pallet<T> {
 		/// Dispatchable to be called by server with privileged account
 		/// dispatch claim
+		#[pallet::call_index(0)]
```
Comment (ibrizsabin) [inline pallets/airdrop/src/lib.rs:236]: are we sure the call index generated prior is equal to what we are hardcoding now? we can make sure by looking into polkadot ui.

File: resources/arctic-launch.json [hunk 2]
```diff
@@ -32,8 +32,8 @@
   },
   "parachains": [
     {
-      "bin": "../target/release/ice-node",
-      "chain": "arctic-dev",
+      "bin": "/Users/abhi/code/ice-substrate/target/release/ice-node",
```
Comment (ibrizsabin) [inline resources/arctic-launch.json:35]: relative path doesn't work here?

**Review by ibrizsabin — APPROVED** [review]

File: pallets/airdrop/src/lib.rs (hunk 1 above)
Comment (abhijeetbhagat) [reply pallets/airdrop/src/lib.rs:236]: thanks sabin for the tip and help on checking the call indices from the ui! i verified all call indices of the methods from 3 pallets and they match the values set in this pr.

File: resources/arctic-launch.json (hunk 2 above)
Comment (abhijeetbhagat) [reply resources/arctic-launch.json]: restored to the main version

**Review by ibrizsabin — APPROVED** [review]

**Review by andrii-kl — APPROVED** [review]

**Pull request merged**